src/test/java/org/cactoos/http/HtWireTest.java

```diff
@@ -122,4 +170,25 @@ private Socket socket() throws IOException {
         Mockito.when(socket.getInputStream()).thenReturn(new DeadInputStream());
         return socket;
     }
+
+    /**
+     * An {@link Input} to GET an HTTP URI.
+     */
+    private static final class GetInput extends InputEnvelope {
```

Comment:
@victornoel a class named like a function... we can do better than that :)

Reply:
@llorllale it's the HTTP GET verb: it's not about getting an input, it's an `Input` implemented with the GET verb. I'm not sure what better can be done, `HttpGetInput`?

Reply:
@victornoel ok